common/TextDifferencer.java

```diff
@@ -111,6 +111,7 @@ private static int getFooterMatchingCharactersLength(
    * Generate matching text changes for the given range. The implementation assumes that all the
    * characters within the given range are equal.
    *
+   * // TODO Check params description
```

Comment:
Can you elaborate? What kind of check?

Reply:
I meant the real parameters of the method and their description in Javadoc are different.

Reply:
So please add that to the TODO:
`// TODO Check params description in Javadoc matches actual method params`